drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/popup/ScenarioSimulationCreationPopupPresenter.java

```diff
@@ -22,28 +22,28 @@
 import org.uberfire.mvp.Command;
 
 @Dependent
-public class ScenarioKogitoCreationPopupPresenter implements ScenarioKogitoCreationPopup.Presenter {
+public class ScenarioSimulationCreationPopupPresenter implements ScenarioSimulationCreationPopup.Presenter {
```

Comment:
Why do you remove `Kogito` from the name of these classes? Are them `Kogito` specific? If not why are they in `kogito-client` module?

Reply:
@danielezonca The points is: inside that package there was some class with `ScenarioSimulation` prefix and other with `ScenarioKogito`. 
Therefore, I used the same one. I choice `ScenarioSimulation`, just because I think it's more appropriate (kogito in already in the package and project name).

Reply:
The name is fine for me, the question is more related to understand if it means that it is not a Kogito specific class or not.

Reply:
Ok, got it. Currently it is used in Kogito only, but technically it could be used in BC too (moving it into a common module). So we can consider it as a Kogito specific class. 
Do you think we should enforce this information definiting a more verbose prefix (eg. ScenarioSimulationKogito or the old one)?

Reply:
I would probably use `Kogito` in class name if it has to be used in Kogito modules only so if these classes can be shared (even if they are not for now) I will stay with `ScenarioSimulation` prefix. Do you agree?

Reply:
@danielezonca Agree. `ScenarioSimulationKogito` in kogito scope. `Rename` function, I'm cooooming! :)

Reply:
@danielezonca Done!